cylc/flow/scripts/cylc_get_suite_version.py

```diff
@@ -32,8 +36,30 @@ def main():
     (options, args) = parser.parse_args()
     suite = args[0]
 
-    pclient = SuiteRuntimeClient(suite, timeout=options.comms_timeout)
-    print(pclient('get_cylc_version'))
+    try:
+        pclient = SuiteRuntimeClient(suite, timeout=options.comms_timeout)
+    except Exception as exc:
+        print(cparse(f'<red>{exc}</red>'))
+        sys.exit(1)
```

Comment:
OOh, looks like this script isn't wrapped with `cylc.flow.terminal.cli_function` for some reason, that will take care of exception formatting.

Reply:
Done.